modules/installation-cloudformation-subnet-localzone.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+[id="installation-cloudformation-subnet-localzone_{context}"]
+= CloudFormation template for the subnet that uses Local Zones
+
+You can use the following CloudFormation template to deploy the subnet that
+you need for your {product-title} cluster that uses Local Zones.
+
+.CloudFormation template for the subnet
+[%collapsible]
+====
+[source,yaml]
+----
+# CloudFormation template used to create Local Zone subnets and dependencies
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for Best Practice VPC with 1-3 AZs
+
+Parameters:
+  ClusterName:
+    Description: ClusterName used to prefix resource names
+    Type: String
+  VpcId:
+    Description: VPC Id
+    Type: String
+  LocalZoneName:
+    Description: Local Zone Name (Example us-east-1-bos-1)
+    Type: String
+  LocalZoneNameShort:
+    Description: Short name for Local Zone used on tag Name (Example bos1)
+    Type: String
+  PublicRouteTableId:
```

Comment:
@mtulio both public and private subnets work well for local zone, I think customer may choose different subnet on demand, how about to change this to `LocalZoneRouteTableId`, `LocalZoneSubnetCidr`, `LocalZoneSubnet` etc.

Reply:
@yunjiang29 I agree for renaming `LocalZoneSubnetCidr`, but renaming the `LocalZoneRouteTableId` for this CloudFormation template could cause confusion, as the RouteTableId is not for Local Zones only, it is created on the VPC CloudFormation template.

Furthermore, to use private subnets on the Local Zones it requires extra steps to associate the Local Zone subntes to one of the **PrivateRouteTable** within the **Regular Region**, as the Local Zones do not support Nat Gateway, so it requires a more complex setup which is not covered here. As we are demonstrating, and testing, the steps to set up **public subnets** when installing in existing VPC, I would suggest to do not mention the steps to use workers on private subnets in LZ.

I think users can customize the template and the deployment, I also think we can further create a guide demonstrating how to do it after GA.

Unless you have tested the scenario with private subnets, I would suggest skipping for now (it is also not covered on the Enhancement).

WDYT?

cc @kalexand-rh @GroceryBoyJr

Reply:
@mtulio Agree. Let's keep it as it is now.

> As we are demonstrating, and testing, the steps to set up **public subnets** when installing in existing VPC, I would suggest to do not mention the steps to use workers on private subnets in LZ.

It makes sense, similar to UPI section , we just demonstrate in these section, user can customize the configuration to meet their requirement. 

> Unless you have tested the scenario with private subnets, I would suggest skipping for now (it is also not covered on the Enhancement).

We had tested Local Zone in private subnet, as mentioned above, using public subnet is reasonable.

cc @kalexand-rh @GroceryBoyJr

Reply:
I'm not making any changes here for GA, but we can talk about adding another guide after GA.